draft-ietf-mls-architecture.md

```diff
@@ -1418,12 +1418,13 @@ MLS provides additional protection regarding secrecy of past messages and future
 messages. These cryptographic security properties are Forward Secrecy (FS) and
 Post-Compromise Security (PCS).
 
-FS means that access to all encrypted traffic history combined with access to
-all current keying material on clients will not defeat the secrecy properties of
-messages older than the oldest key of the compromised client.  Note that this
-means that clients have the extremely important role of deleting appropriate
-keys as soon as they have been used with the expected message, otherwise the
-secrecy of the messages and the security for MLS is considerably weakened.
+FS means that access to all encrypted traffic history combined with
+access to all current keying material on clients will not defeat the
+secrecy properties of messages older than the oldest key of the
+compromised client.  Note that this means that clients the appropriate
+keys as soon as they have been used with the expected message,
+otherwise the secrecy of the messages and the security for MLS is
+considerably weakened.
```

Comment:
```suggestion
compromised client.  Note that this means that clients have to delete the appropriate
keys as soon as they have been used with the expected message,
otherwise the secrecy of the messages and the security for MLS is
considerably weakened.
```

Reply:
Argh. thanks for the fix.